modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/KeyValueStorageUtils.java

```diff
@@ -64,4 +64,12 @@ public static int indexToCompact(long[] keyRevisions, long compactionRevisionInc
     public static String toUtf8String(byte[] bytes) {
         return new String(bytes, UTF_8);
     }
+
+    /** Asserts that the compaction revision is less than the current repository revision. */
```

Comment:
What do you mean by "repository revision"?

Reply:
Typo, fix it, thanks!